api/types/types.go

```diff
@@ -1,8 +1,8 @@
 package types
 
 const (
-	DefaultOAMNS               = "oam-system"
-	DefaultOAMReleaseName      = "vela-core"
+	DefaultOAMNS               = "vela-system"
+	DefaultOAMReleaseName      = "kube"
```

Comment:
I prefer the old OAM release as users might `kube` might be mixed up with kube-xxx releases:) And I like the new OAM namespace.